src/backend/howlongtobeat/utils.ts

```diff
@@ -1,12 +1,35 @@
-import { logError, LogPrefix } from 'backend/logger/logger'
+import { logError, logInfo, LogPrefix } from 'backend/logger/logger'
+import { removeSpecialcharacters } from 'backend/utils'
 import { HowLongToBeatEntry, HowLongToBeatService } from 'howlongtobeat'
+import { howLongToBeatStore } from './electronStores'
 
 export async function getHowLongToBeat(
   title: string
 ): Promise<HowLongToBeatEntry | null> {
   try {
+    title = removeSpecialcharacters(title)
+
+    // check if we have a cached response
+    const cachedResponse = howLongToBeatStore.get(title)
+    if (cachedResponse) {
+      logInfo([`Using cached HowLongToBeat data for ${title}`, ''], {
```

Comment:
Just being nit picky here, but I think this is supposed to be used like:

```ts
logInfo(['Using cached HowLongToBeat data for' title], {
```

Reply:
I think I started one approach, changed it and left that empty string there. Good catch.